src/ol/tilegrid.js

```diff
@@ -129,14 +127,12 @@ function resolutionsFromExtent(extent, opt_maxZoom, opt_tileSize) {
  *     DEFAULT_MAX_ZOOM).
  * @param {number|module:ol/size~Size=} opt_tileSize Tile size (default uses
  *     DEFAULT_TILE_SIZE).
- * @param {module:ol/extent~Extent.Corner=} opt_corner Extent corner (default is
- *     module:ol/extent~Extent.Corner.BOTTOM_LEFT).
+ * @param {module:ol/extent/Corner~Corner=} opt_corner Extent corner (default is `'top-left'`).
```

Comment:
note: the comment was wrong, the default value is really `'top-left'` (see `createForExtent` function)